lab-taylor/lib/server-toggle.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const debug = require('debug')('instaclone:server-toggle');
+const mongoose = require('mongoose');
+
+module.exports = exports = {};
+
+exports.serverOn = function(server, done) {
```

Comment:
There are other ways that you can format this server toggle that you might find works a little more cleanly in your tests.